Kickstarter-iOS/Features/PaymentMethods/Controller/PaymentMethodSettingsViewController.swift

```diff
@@ -207,12 +207,8 @@ internal final class PaymentMethodSettingsViewController: UIViewController,
 
       switch paymentResult {
       case .completed:
-        let paymentDisplayData = PaymentSheetPaymentOptionsDisplayData(
-          image: existingPaymentOption.image,
-          label: existingPaymentOption.label
-        )
         strongSelf.viewModel.inputs
-          .paymentSheetDidAdd(newCard: paymentDisplayData, setupIntent: clientSecret)
+          .paymentSheetDidAdd(setupIntent: clientSecret)
```

Comment:
This data was already unused; `PaymentMethodSettingsViewController` never read `paymentDisplayData`. It just used the `clientSecret`.